### DISCUSSION TITLE
Your Alignment Markers are apparently not being recognized correctly

### DISCUSSION THREAD

**Discussion by tomasenskede** [jomjol/AI-on-the-edge-device#3001] [General]
Your Alignment Markers are apparently not being recognized correctly
https://jomjol.github.io/AI-on-the-edge-device-docs/Alignment/

_Originally posted by @SybexX in https://github.com/jomjol/AI-on-the-edge-device/issues/2990#issuecomment-1998606358_

This looks ok to me... why cant it align?
![image](https://user-images.githubusercontent.com/32595351/313478881-fd74d579-3b5b-4e48-8b05-b7a0c007e642.png)

**Comment by tomasenskede** [comment]
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/32595351/e7bedb62-2b19-46dc-9cb6-3d503d39b268)

**Comment by tomasenskede** [comment]
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/32595351/ffa31978-caf7-42d2-b35c-358b5859c437)

**Comment by tomasenskede** [comment]
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/32595351/b9042997-3a81-424f-9015-3c36ada1d5cc)

**Comment by SybexX** [comment]
The numbers are determined using coordinates, but if the image is not aligned correctly, the coordinates will not be what you expect. If the alignment markers are not far enough apart, are out of focus, are too small, etc., the image cannot be aligned correctly. If you can't find anything sensible on the water meter, you can for example. Attach stickers.

I have marked a few places in the picture that might work
![313478991-ffa31978-caf7-42d2-b35c-358b5859c437](https://github.com/jomjol/AI-on-the-edge-device/assets/587201/d957bcc8-d772-43f2-a052-9ef31978b731)

**Comment by tomasenskede** [comment]
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/32595351/57009ad3-0a9b-4580-bb14-f36b638054e5)
cant get it to work... can I downgrade firmware? it did work with last firmware...

**Reply by SybexX** [reply]
Downgrading is possible, just always use the associated SD card files
https://github.com/jomjol/AI-on-the-edge-device/discussions/2926

**Comment by tomasenskede** [comment]
after downgrading firmware it works again

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/32595351/02ff7f85-eb81-493f-95b8-6378d5e297ec)

**Reply by SybexX** [reply]
In version 15.7.0 some camera settings were added and this changed the image quality.
The image is partly darker .........., which is why this error can occur.
To fix this you should play with the camera settings until you get roughly the same image quality or set the alignment markers differently.
